chapter-05/final/money/presentation.go

```diff
@@ -0,0 +1,28 @@
+package money
+
+import "fmt"
+
+// ParseAmount transforms user-provided amount into a handy Number type.
+func ParseAmount(amount string) (Number, error) {
+	n, err := parseNumber(amount)
+	if err != nil {
+		return Number{}, fmt.Errorf("unable to parse amount: %w", err)
+	}
+
+	return n, nil
+}
+
+// ParseCurrencies transforms user-provided inputs into handy Currency type.
+func ParseCurrencies(from, to string) (Currency, Currency, error) {
```

Comment:
What does it bring us to take both and return both, rather than one by one? This makes testing twice as complex and reduces the reusability of the function.

Reply:
Removed and exposed `ParseCurrency`. Wdyt?